test_runner/fixtures/utils.py

```diff
@@ -484,7 +484,7 @@ def assert_no_errors(log_file, service, allowed_errors):
     for _lineno, error in errors:
         log.info(f"not allowed {service} error: {error.strip()}")
 
-    assert not errors, f"Log errors on {service}: {errors[0]}"
+    assert not errors, f"First log error on {service}: {errors[0]}\nHint: use test_runner/fixtures/pageserver/allowed_errors.py to test any new allowed_error you add"
```

Comment:
Highlighting this amidst the renames.

Reply:
I wonder if it'll be easier to use if we create a script (to run `poerty run test_runner/fixtures/pageserver/allowed_errors.py`), put it into `scripts` and advertise it?